src/services/logging.service.ts

```diff
@@ -0,0 +1,40 @@
+export class Logger
+  implements Pick<Console, "log" | "info" | "warn" | "error">
+{
+  private static instance: Logger;
+  private static _hidden = false;
```

Comment:
I was thinking we'd have a dynamic default based on production vs. development. Testing would be off by default too (the tests for the module would just turn it on). I can't say at this point as there are some lint errors, but I suspect unit tests (if there are logs in the modules under test) will contain logs

Reply:
I mean, you can always just turn on/off by assigning `Logger.debug` to `true`. At any given moment. I did not really know exactly where to tap into `process.env` and I believe we'll be using the logger at development so it's easy to just turn on at the start of the file. Not really sure how to go from here.